config-templates/redhat/odf-remote/4.16/deployment.yaml

```diff
@@ -0,0 +1,582 @@
+apiVersion: v1
+kind: List
+metadata:
+  name: ibm-openshift-container-storage
+  namespace: kube-system
+  annotations:
+    revision: 1
+items:
+  - apiVersion: apiextensions.k8s.io/v1
+    kind: CustomResourceDefinition
+    metadata:
+      annotations:
+        controller-gen.kubebuilder.io/version: v0.12.0
```

Comment:
`controller-gen.kubebuilder.io/version: v0.12.0` is different in ODF Local 4.16
So please confirm of kube-builder is correct here.